README.MD

```diff
@@ -1,14 +1,23 @@
-# di-auth-stub-relying party
+# Relying Party Stub
 
 ## Summary
-This stub allows you to set various paramaters (see [here]("https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/")) and mock calls from an RP.
+This stub allows you to create a mock relying party to initiate a GOV.UK One Login journey.
+It loads a screen where you can set various parameters (see [here]("https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/")), and make calls to the `/authorize` endpoint using these settings.
+Once you have gone through the authentication/identity journey, you will be redirected to the stub, with a page
+displaying the user info returned by the One Login service.
 
 ## Local Setup
 You'll need to set up environment variables to run locally. Make a copy of the `.env.template` file, and rename it `.env`.
+
 The stub is currently set up to connect to the build environment (as sandpit blocks requests from localhost). Once you 
 have a copy of `.env`, you'll need to go into the `build-client-registry` table in DynamoDB, and find the client with ClientName `di-auth-stub-relying-party-build`.
 Update the value of `CLIENT_ID` to the ClientID of this client.
```

Comment:
For the Doc App stub, does the `CLIENT_ID` need to be set to the `di-auth-stub-relying-party-build-app` value?
```suggestion
The stub is currently set up to connect to the build environment (as sandpit blocks requests from localhost). Once you 
have a copy of `.env`, you'll need to go into the `build-client-registry` table in DynamoDB, and find the client with ClientName `di-auth-stub-relying-party-build` or `di-auth-stub-relying-party-build-app` for Doc App.
Update the value of `CLIENT_ID` to the ClientID of this client.
```